src/classes/Commands.ts

```diff
@@ -1295,7 +1295,7 @@ export = class Commands {
         }
 
         this.bot.pricelist
-            .addPrice(params as EntryData, true)
+            .addPrice(params as EntryData, true, params.sku === '5021;6' ? 'manual' : null)
```

Comment:
what if they add keys with autoprice enabled?

Reply:
Should be fine? As it won't update key rate (check the logic in `Pricelist.ts`) but the `src` could misleading.